tictactoe/player.py

```diff
@@ -2,18 +2,25 @@
 import string
 from abc import ABC, abstractmethod
 from random import choice
+from dataclasses import dataclass
 
 from tictactoe import Cell
 from tictactoe.io import ConsoleFrontend
 
 
+@dataclass
+class Turn:
+    row: int
+    column: int
+
+
 class Player(ABC):
     def __init__(self, name=None, frontend=None):
         self.name = name
         self.frontend = frontend
 
     @abstractmethod
-    def get_turn(self, board) -> int:
+    def get_turn(self, board) -> Turn:
```

Comment:
Since we agreed to remove type hints, I thinkwe can remove this, too:

```suggestion
    def get_turn(self, board):
```